modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -9,6 +9,11 @@
 The `install-config.yaml` file requires some additional details.
 Most of the information is teaching the installer and the resulting cluster enough about the available hardware so that it is able to fully manage it.
 
+[NOTE]
+====
+The installer no longer needs the `clusterOSImage` {op-system} image, as the correct image is in the release payload.
```

Comment:
s/installer/installation program

Reply:
```suggestion
The installation program no longer needs the `clusterOSImage` {op-system} image because the correct image is in the release payload.
```

Reply:
Updated as suggested.